src/frontend/src/lib/components/core/Background.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+	import background from '$lib/assets/background.svg';
+	import Img from '$lib/components/ui/Img.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+	import { replaceOisyPlaceholders } from '$lib/utils/i18n.utils';
+</script>
+
+<div class="fixed inset-0 z-[-1] overflow-hidden bg-white">
+	<picture
+		class="absolute left-1/2 top-0 min-w-[1280px] -translate-x-1/2 transform"
+		aria-label={replaceOisyPlaceholders($i18n.core.alt.background)}
+	>
+		<source srcset={background} media="(min-width: 641px)" />
+		<Img src="" />
```

Comment:
Does it throw an error in the console or it's valid to path `""` as `src`?

Reply:
Just to be sure, did you test in Firefox and Safari as well?
In worth case we can add an empty inlined base64 image I guess. Also had a look to the spec indeed not much we can do otherwise.

Reply:
tested right now both of them and the behavior went as expected

But, I can do your suggestion and pass an empty image (I actually tried that too in one of the tests). I thought just that theoretically it is unnecessary, to have a new empty image just to pass it here, but no strong opinion, so WDYT?

Reply:
Let's try with this version